app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -216,17 +216,21 @@ function AddressSearch({
           hint={t('in_person_proofing.body.location.po_search.address_search_hint')}
         />
       </ValidatedField>
-      <SpinnerButton
-        isWide
-        isBig
-        ref={spinnerButtonRef}
-        type="submit"
-        className="margin-y-5"
-        onClick={handleSearch}
-        spinOnClick={false}
-      >
-        {t('in_person_proofing.body.location.po_search.search_button')}
-      </SpinnerButton>
+      <div className="margin-y-5">
+        <SpinnerButton
+          isWide
+          isBig
+          ref={spinnerButtonRef}
+          type="submit"
+          onClick={handleSearch}
+          spinOnClick={false}
+        >
+          {t('in_person_proofing.body.location.po_search.search_button')}
+        </SpinnerButton>
+        <p className="text-primary text-bold">
+          {isLoading && t('in_person_proofing.body.location.po_search.is_searching_message')}
+        </p>
```

Comment:
SpinnerButton has [functionality to do this](https://github.com/18F/identity-idp/blob/aaac9a0bd8e2dfcc81195b0c982ed7d36f961e0b/app/javascript/packages/spinner-button/spinner-button.tsx#L23-L31), I think we should use that

Reply:
eg
```
        <SpinnerButton
          isWide
          isBig
          ref={spinnerButtonRef}
          type="submit"
          onClick={handleSearch}
          spinOnClick={false}
          actionMessage="Searching for Post Offices..."
          longWaitDurationMs={1}
        >

```

Reply:
Looked at the demo here: https://idp.dev.identitysandbox.gov/components/inspect/spinner_button/preview

I think you may need to set `long-wait-duration-ms` in order to make the message visible faster.